### DISCUSSION TITLE
1주차 과제질문입니다.

### DISCUSSION THREAD

**Discussion by Sonheewon** [rinthel/opengl_course#3] [질문 & 답변]
PS C:\Users\COX\Documents\work\cg\cmake_project_example> cmake -Bbuild . -DCMAKE_BUILD_TYPE=Debug
cmake : 'cmake' 용어가 cmdlet, 함수, 스크립트 파일 또는 실행할 수 있는 프로그램 이름으로 인식되지 않습니다. 이름이 정확한지
확인하고 경로가 포함된 경우 경로가 올바른지 검증한 다음 다시 시도하십시오.
위치 줄:1 문자:1
+ cmake -Bbuild . -DCMAKE_BUILD_TYPE=Debug
+ ~~~~~
    + CategoryInfo          : ObjectNotFound: (cmake:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

빌드하려고하면 밑에 처럼 오류가 떠서 인터넷에 검색해서 
powershell부분에서 Select Default Shell 선택 후, Command Prompt 으로 수정하면 된다해서 해봐도 안되고
다른 환경변수 얘기도 나와서 수정해봤는데 안됩니다... 어떻게 해야할까요..?

**Comment by rinthel** [comment]
@Sonheewon `cmake` 경로 설정이 안되어 있어서 생기는 문제 같군요. `cmake`를 설치할 때 path 설정과 관련된 옵션을 체크하는 부분이 있을 거예요. 확인해서 다시 설치하고 진행해보세요.
